ninedot-application/src/main/java/org/sopt36/ninedotserver/ai/usecase/AiSubGoalRecommendationService.java

```diff
@@ -75,7 +74,7 @@ public SubGoalAiResponse fetchAiSubGoalRecommendation(
         User user = userRepository.findById(userId)
             .orElseThrow(() -> new AiException(USER_NOT_FOUND));
 
-        int age = AgeUtil.calculateAgeFromString(user.getBirthday());
+        int age = user.getBirthday().calculateAge();
```

Comment:
P3: AgeUtil 없애고 user 도메인 내부에 메서드 만들어서 풍부하게 만든 것 아주 좋네요 칭찬x100 하겟습니다.

다만! [객체지향 생활체조 원칙](https://velog.io/@inwoo920/%EA%B0%9D%EC%B2%B4%EC%A7%80%ED%96%A5-%EC%83%9D%ED%99%9C-%EC%B2%B4%EC%A1%B0-9%EA%B0%80%EC%A7%80-%EC%9B%90%EC%B9%99%EC%97%90-%EB%A7%9E%EA%B2%8C-Meeting-Service-%EB%A6%AC%ED%8C%A9%ED%86%A0%EB%A7%81%ED%95%98%EA%B8%B0) 중 다섯 번째를 보면, 한 줄에 점을 하나만 찍는다는 원칙이 있습니다. 무조건 점을 하나만 찍으라는 소리는 아니고, 가까운 객체에 메시지를 보내라는 뜻으로 이해하시면 좋을 것 같아요. 그래서 아래와 같은 수정을 제안합니다!

```suggestion
// User.java
public int getAge() {
    if (this.birthday == null) {
            return 0;
        }
    return this.birthday.calculateAge();
}

// Birthday.java => 현재 형태 유지
```